frame/oracle/src/lib.rs

```diff
@@ -73,7 +73,9 @@ pub mod pallet {
 		AccountId32, ArithmeticError, FixedPointNumber, FixedU128, KeyTypeId as CryptoKeyTypeId,
 		PerThing, Percent, RuntimeDebug,
 	};
-	use sp_std::{borrow::ToOwned, fmt::Debug, str, vec, vec::Vec};
+	#[cfg(not(feature = "std"))]
+	use sp_std::vec::Vec;
+	use sp_std::{borrow::ToOwned, fmt::Debug, str, vec};
```

Comment:
Is this change necessary?

Reply:
Added it earlier when I was debugging the `no-std` issue, I'll revert them.